doc/src/quick_start_guide.md

```diff
@@ -23,9 +23,9 @@ regex = "1"
 
 # Add a build-time dependency on the lalrpop library:
 [build-dependencies]
-lalrpop = "0.19.0"
-# If you do not supply your own, external lexer you also need the `lexer` feature
-# lalrpop = { version = "0.18.1", features = ["lexer"] }
+lalrpop = { version = "0.19.0", features = ["lexer"] }
```

Comment:
TBH, we should just make "lexer" a default feature, that would fix the problems at the root.

Reply:
I've attempted to add this though I'm not that familiar with these parts of cargo & rust.